docker-entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+function do_help {
+  echo HELP:
+  echo "Supported commands:"
+  echo "   master              - Start a Kudu Master"
+  echo "   tserver             - Start a Kudu TServer"
+  echo "   help                - print useful information and exit"
+  echo ""
+  echo "Other commands can be specified to run shell commands."
+
+  exit 0
+}
+
+if [ "$1" = 'master' ]; then
+	exec kudu-master -fs_wal_dir /data/kudu-master --logtostderr \
```

Comment:
Good job on the entrypoint script. I like it!

Although I am not sure about putting the parameters "--logtostderr", "--use_hybrid_clock", etc. in this bash script, as now the parameters are splitted in 2 locations (entrypoint.sh and docker run command).